### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by mortent**
Fixing some broken links in the porch install and user guides.

### CODE REVIEW

File: site/guides/porch-installation.md
Comment (martinmaly) [inline site/guides/porch-installation.md:193]: In my experience, some of the links within the site didn't work with the `.md` extension, and many links I encountered were absolute. that said, LGTM if it works though I'd assume `/guides/porch-user-guide` is what we need.

**Review by martinmaly — APPROVED** [review]

File: site/guides/porch-installation.md
Comment (mortent) [reply site/guides/porch-installation.md:193]: I have verified these changes running the site locally. I will verify that this works on the deployed site as well once it is merged.

**Comment by bgrant0607** [comment]
Site build failed with broken links. Real or just rerun?

**Comment by mortent** [comment]
The failure looks like a timeout. Trying a rerun

**Pull request merged**
